packages/host/app/components/operator-mode/container.gts

```diff
@@ -136,6 +139,65 @@ export default class OperatorModeContainer extends Component<Signature> {
     return await this.operatorModeStateService.constructRecentCards();
   });
 
+  @action async onCommand(command: any) {
+    // apply patch
+    if (command.type == 'patch') {
+      for (let item of this.allStackItems) {
+        if ('card' in item && item.card.id == command.id) {
+          await this.setFieldValues(item.card, command.patch.attributes);
+          return await this.save(item);
```

Comment:
this is a little awkward in that the setting of the field values probably already triggers the save via auto save, so the save below is redundant. in general the assignment of field values doesn't happen in this component which is why this double save issue has not come up before. rather field value assignment happens at the edges in the setters that live in the card field components as cards are rendered in the edit format in the stack item components. probably we need to think thru a more cohesive way to assign field values in a way that doesn't collide with autosave

Reply:
Reviewing other approaches is a good plan at this point, I've removed previews but there remains a question of how best to communicate the change to the user - e.g. creating a new card and showing them side by side, overwriting the existing one on an accept, or a custom diff.